lib/acts_as_ar_query.rb

```diff
@@ -0,0 +1,190 @@
+require 'forwardable'
+# allow active record dsl to call legacy find
+class ActsAsArQuery
+  extend Forwardable
+  include Enumerable
```

Comment:
If you have Enumerable, what defines .each?  Does Enumerable even make sense here?